third_party/react-dates/index.js

```diff
@@ -1,2 +1,81 @@
 import 'react-dates/initialize';
 import 'react-dates';
+
+// Production steps of ECMA-262, Edition 6, 22.1.2.1
+if (!Array.from) {
```

Comment:
Where's this code coming from?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

Reply:
The `Array.from`, not the key stuff.

Reply:
Ah sorry, opened the wrong mdn tab 😅
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from